threedi_scenario_downloader/downloader.py

```diff
@@ -102,15 +102,21 @@ def get_scenario_instance(scenario_uuid):
     return scenario_instance
 
 
-def get_scenario_instance_results(scenario_uuid):
+def get_scenario_instance_results(scenario_uuid, subendpoint=None):
     """check if scenario exist"""
     get_scenario_instance(scenario_uuid)
 
+    if subendpoint:
+        url = "{}scenarios/{}/results/{}".format(LIZARD_URL, scenario_uuid, subendpoint)
+    else:
+        url = "{}scenarios/{}/results".format(LIZARD_URL, scenario_uuid)
+
     """return request response with all raster results"""
```

Comment:
Ligt misschien aan github, maar deze docstring lijkt raar in de code te staan.
Aan de andere kant is de docstring "check if scenario exists" niet volledig. => Eén goede docstring aan het bebin v/d functie aub :-)

Reply:
Goeie, ik heb dit aangepast.